src/pages/Transactions/index.tsx

```diff
@@ -85,27 +87,39 @@ const _TransactionsPage = ({
 
   const createButtonText = (): string => {
     if (totalTxCount === 1) {
-      return 'Initiate the transaction.';
+      return formatMessage({ defaultMessage: 'Confirm deposit' });
     }
     if (totalTxCount === remainingTxCount && totalTxCount > 0) {
-      return `Initiate all ${totalTxCount} transactions`;
+      return formatMessage(
+        {
+          defaultMessage: `Confirm all {totalTxCount} deposits`,
```

Comment:
nitpicking: using the word "confirm" may sound like "waiting for 12 block confirmations...". How about using "send"?

Reply:
@ryancreatescopy Thoughts on this? Could go either way... Perhaps "Sign all {} deposits"? It helps remove that feeling of "block confirmations". Agree it's a small issue

Reply:
Agree use of "confirm" could be a bit confusing. I'm find with initiate or send.